base/bucket.go

```diff
@@ -22,6 +22,7 @@ import (
 	memcached "github.com/couchbase/gomemcached/client"
 	sgbucket "github.com/couchbase/sg-bucket"
 	"github.com/couchbaselabs/walrus"
+	"gopkg.in/couchbase/gocbcore.v2"
```

Comment:
Update: gocb currently points to a `gopkg.in` version of gocbcore in order to pin it to a known release when running `go get`.  Hopefully when the [saga of go dependency management](https://blog.gopheracademy.com/advent-2016/saga-go-dependency-management/) improves, the built-in go tooling will make this less clunky.